src/libraries/System.Text.Json/src/System/Text/Json/Serialization/ReferenceHandler.cs

```diff
@@ -22,7 +22,7 @@ public sealed class ReferenceHandling
         /// Metadata properties (`$id`, `$values`, and `$ref`) will not be consumed and therefore will be treated as regular JSON properties.
         /// The metadata properties can map to a real property on the returned object if the property names match, or will be added to the <see cref="JsonExtensionDataAttribute"/> overflow dictionary, if one exists; otherwise, they are ignored.
         /// </remarks>
-        public static ReferenceHandling Default { get; } = new ReferenceHandling(PreserveReferencesHandling.None);
+        public static ReferenceHandler? Default { get; } = null;
```

Comment:
Is it OK to use `null` as the "default" or as the "do not preserve" option? 
As per early discussions with @ahsonkhan, people tend to prefer actually seeing the default option listed, so this would only mask `null` value as the default, instead of having one instance that holds a `ReferenceResolver` that is never used.